examples/connect/docker/docker-compose.yaml

```diff
@@ -10,7 +10,7 @@ services:
     ports:
       - "9464:9464"
       - "4317:4317"
-      - "55681:55681"
+      - "4318:4318"
```

Comment:
The docker image in use is `otel/opentelemetry-collector:0.30.0` which still exposes only the 55681 port by default.
I suspect that this change may break the example.
According to the [collector CHANGELOG](https://github.com/open-telemetry/opentelemetry-collector/blob/main/CHANGELOG.md#v0320-beta) the 4318 port is introduced in version 0.32.0.

Can you please upgrade the otelcol version in this example and in fastify?

Reply:
Great catch! I updated all references to be `otel/opentelemetry-collector:0.38.0` instead since that's what I saw used throughout the repo 🙂